File: app/models/beer.rb
class Beer < ActiveRecord::Base
  attr_accessible :name, :description, :abv, :available, :category, :style, :label_url
	validates_presence_of :name, :description, :category
	validates :name, length: { minimum: 2 }
	validates :category, length: { minimum: 2 }
	validates :description, length: { minimum: 10 }
end

File: app/assets/javascripts/app/models/beer.js
Amber.Beer = DS.Model.extend({
  resourceUrl:'/beers',
  name: DS.attr('string'),
  description: DS.attr('string'),
  abv: DS.attr('string'),
  isOrganic: DS.attr('string'),
  status: DS.attr('string'),
  available: DS.attr('string'),
  category: DS.attr('string'),
  style: DS.attr('string'),
  icon_url: DS.attr('string')
});